apisix/schema_def.lua

```diff
@@ -725,8 +725,19 @@ _M.ssl = {
             default = "server",
             enum = {"server", "client"}
         },
-        cert = certificate_scheme,
-        key = private_key_schema,
+        cert = {
+            oneOf = {
+                { type = "string", minLength = 128, maxLength = 64*1024},
+                -- TODO: uniformly define the schema of secret_uri
+                { type = "string", pattern = "^\\$secret://"}
```

Comment:
Does this could work?
```
        cert = {
            oneOf = {
                certificate_scheme,
                { type = "string", pattern = "^\\$secret://"}

```